File: L006/Arrays/minEle.cpp
private static int minElement(int[] arr){
        int minEl = (int) 1e8;

        for(int el: arr){
            if(el < minEl) minEl = el;
        }

        return minEl;
    }


     //  public int minElement(int arr[]){
    //     int minEle = -1e8;
    //     for(int i=0;i<arr.length;i++){
    //         if(a[i]<minEle)
    //          minEle = a[i]; 
    //     }
    //     return minEle;
    // }

File: L006/Arrays/minEle.java
import java.util.Scanner;

public class minEle{

    public static Scanner sc = new Scanner(System.in);

    public static int minEle_(int[] arr){

        int minElem = (int) 1e8;
        for(int ele:arr){
            if(ele < minElem) minElem = ele;
        }
        return minElem;
    }

    public static void main(String[] args){

        int n = sc.nextInt();

        int[] arr = new int[n];

        for(int i=0;i<arr.length;i++){
            arr[i] = sc.nextInt();
        }

        System.out.println("Min element is : " + minEle_(arr));
    }
}
